swift-sdk/IterableInboxViewController.swift

```diff
@@ -89,13 +93,15 @@ open class IterableInboxViewController: UITableViewController {
 
     private let iterableCellNibName = "IterableInboxCell"
     
-    private var viewModels = [InboxMessageViewModel]()
-
+    private lazy var diffCalculator: TableViewDiffCalculator<Int, InboxMessageViewModel>? = {
+        TableViewDiffCalculator(tableView: self.tableView, initialSectionedValues: SectionedValues([]))
```

Comment:
If we only have one section, it might be easier/cleaner to use `SingleSectionTableViewDiffCalculator` instead.